map/src/util/Utils.js

```diff
@@ -130,6 +155,7 @@ export function prepareFileName(filename) {
         return new TextDecoder().decode(truncated);
     };
 
+    const newlineRe = /\n/g;
```

Comment:
Add comments about all regex (what does he do and why)

Reply:
details added